src/core/pluginRunner.ts

```diff
@@ -7,22 +7,16 @@ export async function RunPlugins(
   args,
   pluginCounter?
 ) {
-  if (!pluginCounter) {
-    pluginCounter = 0;
-  }
-  if (!job) {
-    return true;
-  }
+  if (!pluginCounter) pluginCounter = 0;
+  if (!job) return true;
```

Comment:
my coding style is never to do such "oneliners". always use braces for code blocks.

it keeps a consistent style, avoids bugs.

if you feel like the optimization makes code shorter, perhaps the function is too long, has too many responsibilities, and should be refactored by [method extraction](https://refactoring.guru/extract-method), etc.

also, for code minifying there's automated tools, no need to obfuscate code manually.

my 2ç

Reply:
Thanks for your feedback!

I generally disagree, coming from a Ruby background.  In my experience, folks tend to be able to be able to parse the logic of one liners more easily, as long as they are simple (return or set a value).   The bounds of this conversation and the complexity of what would be allowed in 1-liners is checked be Prettier (Linter), which is comfortable with "short" 1-liners.

Reply:
Spotted micro optimization, these lines should be swapped:

```node
  if (!pluginCounter) pluginCounter = 0;
  if (!job) return true;
```

there's no point sanitize `pluginCounter` if next line returns from the method.

Reply:
Exactly the kind of thing that's easier to see with these one-liners 👍 

Lines swapped @ https://github.com/actionhero/node-resque/commit/664b58df2877197668edbbacc4779af8db0eb770

Reply:
thought to share this: https://blog.codecentric.de/en/2014/02/curly-braces/